genie-agent/src/main/java/com/netflix/genie/agent/execution/process/impl/JobProcessManagerImpl.java

```diff
@@ -170,11 +166,11 @@ public void kill(final KillService.KillSource source) {
      */
     @Override
     public JobProcessResult waitFor() throws InterruptedException {
-        if (!this.launched.get()) {
+        if (!launched.get()) {
```

Comment:
I personally perfer to leave `this` in for member variables so that if I happen to look at the code in something that doesn't include syntax highlighting I'll know easily where it came from. I guess another way would be to do what some people do with `_` or `m` prefix to member variables but for consistency I just put `this.`